src/main.rs

```diff
@@ -204,8 +207,13 @@ fn run() -> Result<()> {
                                  .value_name("COMMIT-MESSAGE")
                                  .help("Commit message that will be used on import")
                                  .default_value("cobalt site import")
-                                 .takes_value(true)))
-        .get_matches();
+                                 .takes_value(true)));
+
+    #[cfg(all(feature="syntax-highlight", not(windows)))]
+    let app_cli = app_cli.subcommand(SubCommand::with_name("syntax-themes")
+                        .about("list available themes"));
```

Comment:
Would it simplify things if we went the "null object" route and made `list_themes` return nothing on Windows?

It let's us isolate the platform checks to just a few areas